protocol/x/subaccounts/types/expected_keepers.go

```diff
@@ -88,6 +89,7 @@ type BankKeeper interface {
 		amt sdk.Coins,
 	) error
 	SendCoinsFromModuleToModule(ctx context.Context, senderModule, recipientModule string, amt sdk.Coins) error
+	SendCoins(ctx context.Context, fromAddr, toAddr sdk.AccAddress, amt sdk.Coins) error
```

Comment:
honestly no good reason - it's not very clean code. Then just made helpers to get collateral pool return the accAddress object instead of a string and therefore need to use sendCoins rather than sendCoinsFromModuleToModule

Reply:
i see so more of a refactoring